WMF Framework/NSManagedObjectContext+History.swift

```diff
@@ -17,6 +17,10 @@ extension NSManagedObjectContext {
             }
             articles = try fetch(request)
         }
+        
+        Task {
+            try await WMFCategoriesDataController().deleteAllCategories()
+        }
```

Comment:
We actually may be okay without this - I see that in WMFHistoryViewController, we already [call](https://github.com/wikimedia/wikipedia-ios/blob/bbaeca39a937c2aeb6fb80f6b814db68ba307e2a/Wikipedia/Code/HistoryViewController.swift#L298) a method `deleteAllPageViewsAndCategories` when the user deletes all. Can you remove this new method and check to see if it still works as expected?